vuln/npm/new.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": 484,
```

Comment:
funny thing about the ids - they're hand-picked at the moment which is a hassle that we're trying to solve by automatically generating random ids :). Right now just pick something that doesn't exist yet and no open PR picked it up before you.

Reply:
sounds good, I'll go with `485` which looks next in line

Reply:
random idea:  if you are committed to using PRs as the means of filling in these files, perhaps using the PR#.  Though its a bit of a chicken and egg problem b/c the PR has to exist before the number is known for certain.

Regardless, simple manual assignment seems to work ok for the volume your dealing with currently.